pkg/cli-runtime/survey.go

```diff
@@ -0,0 +1,17 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/vito/go-interact/interact"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/cli-runtime/printer"
+)
+
+// NewConfirmSurvey create a survey asking for [yN] confirmation when `Resolve` is called
+func NewConfirmSurvey(c *Config, format string, a ...any) *interact.Interaction {
+	i := interact.NewInteraction(fmt.Sprintf("%s %s", printer.Ssuccessf("?"), printer.Sboldf(fmt.Sprintf(format, a...))))
```

Comment:
why this interaction call vs choice func? 
ref: https://github.com/vito/go-interact/blob/master/interact/choice.go
Want to understand your reasoning. can we use choice with `y` and `N` as options?

Reply:
Choice will show the options like:
```console
1: y
2: n
```
without case validation, `NewConfirmSurvey`, when no *options* are used defaults to bolean validation with `y` & `n` values promt with case validation